grid_map/src/GridMapRosConverter.cpp

```diff
@@ -383,6 +383,61 @@ bool GridMapRosConverter::addColorLayerFromImage(const sensor_msgs::Image& image
   return true;
 }
 
+
+bool GridMapRosConverter::toCvImage(grid_map::GridMap& gridMap, const std::string& layer,
+                                    cv::Mat& cvImage, float maxHeight, float minHeight)
+{
+  if (gridMap.getSize()(0) > 0 && gridMap.getSize()(1) > 0) {
+    // Initialize blank image:
+    cvImage = cv::Mat::zeros(gridMap.getSize()(0), gridMap.getSize()(1), CV_8UC4);
+  } else {
+    ROS_ERROR("Invalid grid map?");
+    return false;
+  }
+
+  unsigned int depth = std::pow(2,8) - 1;
+
+  // Find upper and lower values.
+  // This should be replaced, possibly in GridMap.cpp or with cv::convertScaleAbs
+  // (can't use .maxCoeff because of nans).
+  float lowerHeight = 100.0;
```

Comment:
What are these magical numbers? I added the `Clamp` Eigen functor which could help here.